Add e2e tests, refs #13473

jraddaoui opened the pull request:
- Setup integration tests using Cypress:
  - Add Cypress and a couple of commands as NPM dev. dependencies.
  - Add tests for:
    - Login.
    - CSV import maintains siblings order.
    - Basic description creation and deletion.
    - Stop words searches.
- Add GA workflow to run e2e tests in the CI process:
  - Setup workflow runner similar to the installation docs.
  - Run tests only over Ubuntu 20.04 and PHP 7.4, using Electron, Chrome
    and Firefox browsers.

sbreker reviewed (approved):
Looks great @jraddaoui !

jraddaoui committed:
Add e2e tests, refs #13473

- Setup integration tests using Cypress:
  - Add Cypress and a couple of commands as NPM dev. dependencies.
  - Add tests for:
    - Login.
    - CSV import maintains siblings order.
    - Basic description creation and deletion.
    - Stop words searches.
- Add GA workflow to run e2e tests in the CI process:
  - Setup workflow runner similar to the installation docs.
  - Run tests only over Ubuntu 20.04 and PHP 7.4, using Electron, Chrome
    and Firefox browsers.
